test/unit/application_validator_test.rb

```diff
@@ -32,7 +32,10 @@ class ApplicationValidatorTest < Minitest::Test
 
     test "check_autoload_path_cache fails on extraneous config load paths" do
       use_template(:minimal)
-      ApplicationLoadPaths.expects(:extract_relevant_paths).returns([])
+      ApplicationLoadPaths
+        .expects(:extract_relevant_paths)
+        .with { |root_path| root_path.match?(%r{tmp\/fixtures}) }
```

Comment:
Consider using [`ApplicationFixtureHelper.app_dir`](https://github.com/Shopify/packwerk/blob/19d40eadd70758eff6e9fbbab8d3ec69c84a03a4/test/support/application_fixture_helper.rb#L23) to get the absolute directory of your temporary app:
```ruby
.with { |root_path| root_path.match?(%r{#{app_dir}}) } # I have not tried this
```

Reply:
Ah cool, thanks @rochlefebvre 👍  Have moved it to using `.with(app_dir)`